src/pages/FormPage.js

```diff
@@ -126,6 +162,7 @@ const Textarea = styled.textarea`
   font-size: 1.1em;
   line-height: 1.5em;
   border-radius: 4px;
+  resize: none;
```

Comment:
Really love to see this - totally underrated style feature

Reply:
totally!!